src/vivarium_csu_alzheimers/constants/data_values.py

```diff
@@ -32,3 +34,24 @@
 SCREENING_SCALE_UP_DIFFERENCE = (
     SCREENING_SCALE_UP_GOAL_COVERAGE - PROBABILITY_ATTENDING_SCREENING_START_MEAN
 )
+
+
+# Gamma distribution parameters from Nathaniel's nb
+# Fix mean at midpoint of interval [3.5, 4]
+# Adjust variance until about 90% of probability lies in interval (variance = .03)
+
+# Use method of moments to get shape and rate parameters
+GAMMA_SHAPE = 3.75**2 / 0.03  # shape parameter alpha
+RATE = 3.75 / 0.03  # rate parameter lambda
+
+# Convert rate to scale for scipy
+GAMMA_SCALE = 1 / RATE
+
+GAMMA_GAMMA_DIST = scipy.stats.gamma(GAMMA_SHAPE, scale=GAMMA_SCALE)
```

Comment:
consider naming this to make it clear that it is  specifically for the BBBM transition rate